src/common.ts

```diff
@@ -226,26 +226,26 @@ export class AKE3DH implements AKEFn {
         this.Nsk = Nsk
     }
 
-    async deriveAuthKeyPair(seed: Uint8Array): Promise<AKEKeyPair> {
+    async deriveDHKeyPair(seed: Uint8Array): Promise<AKEKeyPair> {
         const keypair = await deriveKeyPair(
             Oprf.Mode.OPRF,
             this.suiteID,
             seed,
-            Uint8Array.from(LABELS.OPAQUE_DeriveAuthKeyPair)
+            Uint8Array.from(LABELS.OPAQUE_DeriveDHKeyPair)
         )
         return { private_key: keypair.privateKey, public_key: keypair.publicKey }
     }
 
-    recoverPublicKey(private_key: Uint8Array): AKEKeyPair {
-        const public_key = generatePublicKey(this.suiteID, private_key)
-        return { private_key, public_key }
-    }
-
-    async generateAuthKeyPair(): Promise<AKEExportKeyPair> {
-        const keypair = this.recoverPublicKey(await randomPrivateKey(this.suiteID))
-        return {
-            private_key: Array.from(keypair.private_key),
-            public_key: Array.from(keypair.public_key)
-        }
-    }
+    // recoverPublicKey(private_key: Uint8Array): AKEKeyPair {
+    //     const public_key = generatePublicKey(this.suiteID, private_key)
+    //     return { private_key, public_key }
+    // }
+
+    // async generateAuthKeyPair(): Promise<AKEExportKeyPair> {
+    //     const keypair = this.recoverPublicKey(await randomPrivateKey(this.suiteID))
+    //     return {
+    //         private_key: Array.from(keypair.private_key),
+    //         public_key: Array.from(keypair.public_key)
+    //     }
+    // }
```

Comment:
do you mean to remove this, or leave it commented out?
if it's meant to be commented out,. add a comment as to why

Reply:
ups